sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/api/Azure.ResourceManager.DesktopVirtualization.netstandard2.0.cs

```diff
@@ -898,25 +839,66 @@ public partial class ScalingSchedule
         public static bool operator !=(Azure.ResourceManager.DesktopVirtualization.Models.ScalingScheduleDaysOfWeekItem left, Azure.ResourceManager.DesktopVirtualization.Models.ScalingScheduleDaysOfWeekItem right) { throw null; }
         public override string ToString() { throw null; }
     }
-    public partial class SendMessage
-    {
-        public SendMessage() { }
-        public string MessageBody { get { throw null; } set { } }
-        public string MessageTitle { get { throw null; } set { } }
-    }
     public partial class SessionHostHealthCheckFailureDetails
     {
         internal SessionHostHealthCheckFailureDetails() { }
         public int? ErrorCode { get { throw null; } }
         public System.DateTimeOffset? LastHealthCheckOn { get { throw null; } }
         public string Message { get { throw null; } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct SessionHostHealthCheckName : System.IEquatable<Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public SessionHostHealthCheckName(string value) { throw null; }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName AppAttachHealthCheck { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName DomainJoinedCheck { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName DomainReachable { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName DomainTrustCheck { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName FSLogixHealthCheck { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName MetaDataServiceCheck { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName MonitoringAgentCheck { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName SupportedEncryptionCheck { get { throw null; } }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.SessionHostHealthCheckName SxSStackListenerCheck { get { throw null; } }
```

Comment:
SxS is one acromyn (Side by Side), the case should probably be `SxsStackListenerCheck `